### PR TITLE
hide files for base image stage

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [x] Python type annotations added to new code
- n/a JSON/YAML configuration changes are updated in the relevant schema
- n/a Changes to metadata also update the documentation for the metadata
- n/a Pull request has a link to an osbs-docs PR for user documentation updates
- n/a New feature can be disabled from a configuration file

### CODE REVIEW

**Comment by mkosiarc** [comment]
Could you please add more description to the commit message?

**Review by ben-alkov — APPROVED** [review]

File: atomic_reactor/plugins/hide_files.py [hunk 1]
```diff
@@ -55,26 +61,46 @@ def run(self):
...
 
+    def _get_custom_image_index(self) -> List[int]:
+        """get indexes for baseimage scratch stages"""
+        custom_image_index = []
+        df_images = self.workflow.data.dockerfile_images.original_parents
+
+        self.log.info("df images: '%s'", df_images)
+
+        for idx, img in enumerate(reversed(df_images)):
+            self.log.info("parent_image '%s' : '%s'", idx, str(img))
```
Comment (mkosiarc) [inline atomic_reactor/plugins/hide_files.py:76]: is this important info to log, or log at info level?

Comment (mkosiarc) [reply atomic_reactor/plugins/hide_files.py:76]: and we have three log messages in this function

Comment (rcerven) [reply atomic_reactor/plugins/hide_files.py:76]: they were just for debugging removing them

**Comment by rcerven** [comment]
> Could you please add more description to the commit message?

added

**Review by mkosiarc — APPROVED** [review]

**Comment by rcerven** [comment]
rn updated

**Pull request merged**